infra/stream/Formatter.cpp

```diff
@@ -22,17 +22,17 @@ namespace infra
         {
             switch (format[i])
             {
-            case '<':
-                align = FormatAlign::left;
-                break;
-            case '^':
-                align = FormatAlign::center;
-                break;
-            case '>':
-                align = FormatAlign::right;
-                break;
-            default:
-                break;
+                case '<':
```

Comment:
I absolutely love this! Finally, a tool that understands that the contents of switches need two separate levels of indentation! Take that, Visual Studio! Woohoo!